.travis.yml

```diff
@@ -1,17 +1,29 @@
+sudo: false
+
 os:
   - linux
   - osx
 
 language: d
 
 d:
-  - ldc2
+  - ldc
   - dmd
 
-before_install:
-  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then sudo apt-get install python-tk; fi
-  - sudo pip install matplotlib
+env:
+  - PYTHON_VER=2
+  - PYTHON_VER=3
 
+addons:
+  apt:
+    packages:
+    - python-tk
+    - python3-tk
+    - python-matplotlib
+    - python3-matplotlib
+
+before_install:
+  - if [ $TRAVIS_OS_NAME = osx ]; then brew install python$PYTHON_VER; brew install homebrew/science/matplotlib; fi
```

Comment:
Travis seems to fail due to this line. It seems better to specify Python's version by using `pyenv`, but I have no experience on travis with `pyenv`. Can you solve this?